.circleci/workflows.yml

```diff
@@ -1097,6 +1141,18 @@ commands:
           paths:
             - cypress/cypress.zip
 
+  trigger-publish-binary-pipeline:
+    steps:
+      - run:
+          name: "Trigger pipeline in publish-binary project"
+          command: |
+            source ./scripts/ensure-node.sh
+            yarn ts-node ./scripts/binary/trigger-publish-binary-pipeline.ts
```

Comment:
It looks like the convention is just using js for build scripts - probably because we don't want to assume `typescript` is available. Either way this script is so simple, should we just make it `.js`? You could do `// @ts-check` for good measure.